src/ADAL.PCL.iOS/PlatformParameters.cs

```diff
@@ -44,11 +44,20 @@ private PlatformParameters()
             SkipBroker = true;
         }
 
+        /// <summary>
+        /// Additional parameters used in acquiring user's authorization
+        /// </summary>
+        /// <param name="callerViewController">UIViewController instance</param>
         public PlatformParameters(UIViewController callerViewController):this()
         {
             this.CallerViewController = callerViewController;
         }
 
+        /// <summary>
+        /// Additional parameters used in acquiring user's authorization
+        /// </summary>
+        /// <param name="callerViewController">UIViewController instance</param>
+        /// <param name="skipBroker">skips calling to broker if broker is present. true, by default</param>
         public PlatformParameters(UIViewController callerViewController, bool skipBroker):this(callerViewController)
         {
             SkipBroker = skipBroker;
```

Comment:
non related, but adal already change it useBroker, and the default value is false.